### ISSUE TITLE
[feature-request] one UUID for all vless vmess trojan or ws tcp grpc ...

### ISSUE THREAD

**Issue by AliM1988** [diditra/x-ui#13]
### Welcome

- [X] Yes, I'm using the latest major release. Only such installations are supported.
- [X] Yes, I'm using the supported system. Only such systems are supported.
- [X] Yes, I have read all WIKI document,nothing can help me in my problem.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, log, etc).

### Description of the problem,screencshot would be good

I see new panel named "Hiddify" can handle multiple protocols (vless vmess trojan) and multiple transmission (ws tcp grpc) within one UUID. It is the major lack of X-UI. So. is it possible to implement ?

### Version of x-ui

-

### x-ui log or xray log

-

**Comment by diditra** [comment]
Yes, and it's one of the things I want to get started on as soon as possible. So I'll probably delay adding Reality and instead will do this in May.
